Sprint-1/implement/max.js

```diff
@@ -1,4 +1,23 @@
 function findMax(elements) {
+  const numericValues = elements.filter(
+    (element) => typeof element === "number" && !isNaN(element)
+  );
+  let maxNumber = numericValues[0];
+  //console.log(numericValues);
+  if (
+    elements.length === 0 ||
+    elements.every((element) => typeof element != "number")
+  ) {
+    return "-Infinity";
```

Comment:
In JavaScript, `-Infinity` is a numeric value and **not** a `String`.  You should here have:
`return -Infinity;`